src/main/java/org/apache/lucene/analysis/AbstractBookTokenFilter.java

```diff
@@ -17,7 +17,7 @@
  * © CrossWire Bible Society, 2008 - 2016
  *
  */
-package org.crosswire.jsword.index.lucene.analysis;
+package org.apache.lucene.analysis;
```

Comment:
why are these moved away from our namespace (org.crosswire)?

Reply:
Because I needed to access some protected methods of Lucene classes in order to implement AbstractBookAnalyzer. Access to protected is only allowed in the same namespace.

Reply:
Hmm then solution is somewhat hacky. Options to consider:
1. Fork lucene analysis lib and remove protected from that particular class (and make upstream PR). Use fork while it is needed.
2. Maybe protected is for a reason? Use some other way if lib author suggest something.
3. Accept hackyness and just leave it like this.

Reply:
I found out that there's public abtract class StopWordAnalyzerBase that probably could be used as a base class. At least that is the baseclass within lucene core lib that is used for per-language classes there.

Reply:
Question arises are all custom per-language analyzers still really needed or could we simplify code by using analyzers from lucene core directly. 

AbstractBookAnalyzer carries book info and it is passed to some filter classes, but any of those does not seem to use that information. I am having a feeling that all that could be simplified greatly.

Reply:
I agree, I'll look into it